src/main/java/com/fcgl/madrid/forum/model/Post.java

```diff
@@ -112,4 +114,12 @@ public void setCityId(int cityId) {
     public void setComment(List<Comment> comment) {
         this.comment = comment;
     }
+
+    public int getUserId() {
+        return userId;
+    }
+
+    public void setUserId(int userId) {
```

Comment:
Think this should be removed as the user who created the comment shouldn't change

Reply:
Same as above

Reply:
good call